### DISCUSSION TITLE
Export binary budget output with multiple package instances

### DISCUSSION THREAD

**Discussion by gschampion** [modflowpy/flopy#2122] [Q&A] [answered]
I have a MF6 model (DIS grid) with multiple instances of the DRN package that each send some flow to MVR. I want to distinguish the flows from the different instances, and this works nicely in the list output and in Zone Budget. I want to be able to extract the cell-by-cell fluxes in a usable format, however.

I used flopy.export.utils.output_helper to export both a NetCDF file and shapefiles for individual layers. Each of these only included the first instance of 'drn' and 'drn_to_mvr'. For example,

import xarray as xr
ds = xr.open_dataset('cbc.nc',engine = 'netcdf4')
list(ds.data_vars)
['crs', 'elevation', 'x_proj', 'y_proj', 'latitude_longitude', 'delc', 'delr', 'VerticalTransform', 'flow_ja_face', 'wel', 'drn', 'drn_to_mvr', 'riv', 'rcha', 'rch', 'evt', 'chd', 'sfr']

I mapped the locations of the drn and drn_to_mvr with values, and they only include the first of several drn and drn_to_mvr records in the cell-by-cell flow file. Is there another way to do this, or do I have to consolidate all the DRN instances into one?

Thanks,
Glen

**Comment by christianlangevin** [comment, marked as answer]
Hey @gschampion, there is a pull request that addresses this issue (#2050) and allows you pull out flows by package name.  Let me see if I can get this merged into flopy.

**Reply by gschampion** [reply]
Ok, thanks, I didn't search there.
